nx.json

```diff
@@ -197,9 +197,6 @@
       ],
       "outputs": [
         "{projectRoot}/dist"
-      ],
-      "dependsOn": [
```

Comment:
I removed the one above in another PR :'(
Would avoid a conflict/mistake to align :D